contentcuration/contentcuration/frontend/shared/views/files/Thumbnail.vue

```diff
@@ -115,10 +89,6 @@
         required: false,
         default: false,
       },
-      isEmpty: {
```

Comment:
Relates to (1) of the main comment, but haven't checked visually in this case. `isEmpty` property is removed here but from the code, I can see that it is still actively used by some features that import `files/Thumbnail.vue`, [for example here in the clipboard](https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/ContentNode.vue#L44). It'd be good to make sure that it can be really deprecated from other features as well (and if yes, to clean up all references to it).